app/domain/etl/ga/internal_search_processor.rb

```diff
@@ -1,3 +1,4 @@
+require_dependency 'concerns/trace_and_recoverable'
```

Comment:
If you think this is the right change, then that's fine :+1: 

But as I don't know much about Rails and Ruby's behaviour in this regard, I'd be interested to know why Rails autoloading isn't handling this, and why this is `require_dependency` rather than just `require`?